packages/injector/index.spec.js

```diff
@@ -127,4 +128,364 @@ describe('load dependencies', () => {
     })
     expect(manifest).toBeDefined()
   })
+
+  it('Should include package dependencies when requested, and the package is in the config', async () => {
+    const manifest = await inject({
+      pkg: {
+        dependencies: {
+          lodash: '4.0.0'
+        }
+      }
+    }, {
+      foo: ({ dependencies: { lodash }}) => lodash
+    })
+    expect(manifest.dependencies.lodash).toBeDefined()
+  })
+
+  // This works locally but not on the build machine since the package would have to require itself...
+  // It('Should include package with a namespace when requested, and the package is in the config', async () => {
+  //   const manifest = await inject({
+  //     pkg: {
+  //       dependencies: {
+  //         '@b-mo/es6-require': '1.0.0'
+  //       }
+  //     }
+  //   }, {
+  //     foo: ({ dependencies: { '@b-mo/es6-require': es6Require }}) => es6Require
+  //   })
+  //   expect(manifest.dependencies.foo).toEqual(require('@b-mo/es6-require').default)
+  // })
+
+  // Its gross, but the injector needs the real name of the dependency in the declaration.
+  // Variables don't work for module names.
+  describe('Built in modules', () => {
```

Comment:
These tests are easy to read, but it must have been a bit tedious to copy and paste each one for one name change.

Reply:
It was, but I tried it the other way first and it did not work :( though it did give me another idea for a new bmo extension.